adapters/infoawarebidder.go

```diff
@@ -172,5 +195,123 @@ func parseBidderInfo(info config.BidderInfo) parsedBidderInfo {
 		parsedInfo.dooh.enabled = true
 		parsedInfo.dooh.banner, parsedInfo.dooh.video, parsedInfo.dooh.audio, parsedInfo.dooh.native = parseAllowedTypes(info.Capabilities.DOOH.MediaTypes)
 	}
+	parsedInfo.multiformat = IsMultiFormatSupported(info)
+
 	return parsedInfo
 }
+
+// AdjustImpForPreferredMediaType modifies the given impression to retain only the preferred media type.
+// It returns the updated impression and any error encountered during the adjustment process.
+func AdjustImpForPreferredMediaType(imp *openrtb2.Imp, preferredMediaType openrtb_ext.BidType) (bool, []error) {
```

Comment:
`AdjustImpForPreferredMediaType` is not called outside this package so there's no need to export it by capitalizing it's first letter.